README.md

```diff
@@ -1,96 +1,120 @@
-    COPY VOCABULARY FROM 'C:\projects\Athena\src\test\resources\testdata\VOCABULARY.csv' WITH DELIMITER E'\t' CSV HEADER QUOTE E'\b' ;
-    COPY RELATIONSHIP FROM 'C:\projects\Athena\src\test\resources\testdata\RELATIONSHIP.csv' WITH DELIMITER E'\t' CSV HEADER QUOTE E'\b' ;
-    COPY CONCEPT_CLASS FROM 'C:\projects\Athena\src\test\resources\testdata\CONCEPT_CLASS.csv' WITH DELIMITER E'\t' CSV HEADER QUOTE E'\b' ;
-    COPY DOMAIN FROM 'C:\projects\Athena\src\test\resources\testdata\DOMAIN.csv' WITH DELIMITER E'\t' CSV HEADER QUOTE E'\b' ;
-    REFRESH MATERIALIZED VIEW concepts_view;`
-
-4. Start SOLR:
-    + in `solrconfig.xml` in `<requestHandler name="/dataimport" class="solr.DataImportHandler">` section change 
-   `<str name="url">jdbc:postgresql://localhost:5432/athena_cdm_v5</str>` to `<str name="url">jdbc:postgresql://localhost:5432/athena_cdm_v5_test</str>`
-   
-    + Launch reindex via web-console
-    + Restart SOLR container if it is running in Docker
-   
-5. Launch tests
\ No newline at end of file
+#1 Phrase search
+
+Search provides the ability to search by phrase. All results are sorted by default according to the following criteria:
+
+ -full phrase match 
+- concepts contain all the words from the search phrase
+- result based on two parameters, the number of searched words in the result and importance of each word (importance is calculated for each word, the words that are rearer among all documents are more important)
+
+Example:
+
+Search phrase: **honey eats pooh**
+
+Name | sort priority explanation |
+--- | ---|
+honey eats Pooh | full match  |
+Pooh eats honey | all words |
+Pooh eats raspberries and honey | 3 words |
+Pooh steals honey | 2 words |
+pooh eats |2 word |
+pooh eats pooh |2 word |
+pooh eats nothing |2 words |
+Pooh eats raspberries |2 words |
+Pooh eats raspberries and me | 2 words |
+Piglet hates honey | 1 words |
+pooh | 1 words |
+Pooh | 1 words |
+pooo | 1 words |
+
+
+NB: the search goes through all concept fields, but the highest priority is given to CONCEPT_NAME and CONCEPT_CODE
+
+#2 Exact search
+
+Using quotation marks forces an exact-match search. 
+
+For an exact search, the following conditions are met
+- the word must be present
+- not case sensitive, the number of spaces between words does not matter
+- stemming is disabled(the word/words must be present exactly as it is in quotation marks )
+
+Example 1:
+
+Search phrase: **"Pooh eats honey"**
+
+Name |
+--- | 
+Pooh eats honey |
+
+Example 1:
+
+Search phrase:  **"Pooh eats" honey**
+
+Name |
+--- |
+Pooh eats honey |
+Pooh eats raspberries and honey |
+pooh eats |
+pooh eats nothing |
+pooh eats pooh |
+Pooh eats raspberries |
+Pooh eats raspberries and me |
+
+
+#3 Special symbols
+
+For special symbols, the following conditions are met
+- If the character is not part of the word, it is ignored.
```

Comment:
What does the `part of the word` mean? What are the `special symbols` except for the ones mentioned in the next line?

Reply:
Overall clarification would be good